main.go

```diff
@@ -146,16 +129,18 @@ func main() {
 
 	timeout, err := strconv.ParseInt(inputs.BootTimeout, 10, 64)
```

Comment:
stepconf package can parse int values: https://github.com/bitrise-io/go-steputils/blob/master/stepconf/example_test.go#L14, what about letting stepconf doing this int parsing?